src/server/async/netlink_server.cc

```diff
@@ -0,0 +1,337 @@
+/** Copyright 2020-2022 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#if BUILD_NETLINK_SERVER == ON
```

Comment:
Use

```
#if defined(BUILD_NETLINK_SERVER) && BUILD_NETLINK_SERVER
#endif
```

and add the following definition in CMakeLists.txt

```
if (BUILD_NETLINK_SERVER)
    target_compile_options(vineyardd PRIVATE -DBUILD_NETLINK_SERVER=1)
endif()
```

Reply:
OK